client/src/App.js

```diff
@@ -17,6 +18,9 @@ function App() {
     saveToLocal('members', orderedMembers);
   }, [orderedMembers]);
 
+  const location = useLocation();
+  const member = location?.state?.member ?? null;
```

Comment:
Maybe `memberDetails` or something like that? You have a lot of similar names with `member`, `members`, ... .

Reply:
You are right, the many members are a little confusing. I will try to find a new name here :-)